libs/experimental/langchain_experimental/sql/base.py

```diff
@@ -140,6 +141,8 @@ def _call(
                     sql_cmd
                 )  # output: sql generation (no checker)
                 intermediate_steps.append({"sql_cmd": sql_cmd})  # input: sql exec
+                if SQL_QUERY in sql_cmd:
+                    sql_cmd = sql_cmd.split(SQL_QUERY)[1]
```

Comment:
should we also .strip to be safe (`sql_cmd.split(SQL_QUERY)[1].strip()`), imagine there would often be a space first character